Main.elm

```diff
@@ -244,14 +282,18 @@ view model =
 
 subscriptions : Model -> Sub Msg
 subscriptions model =
-    Sub.batch
-        [ newData NewData
-        , newError NewError
-        , dataSaved DataSaved
-        , dataNotSaved DataNotSaved
-        , input SetData
-        , Time.every (Time.millisecond * 200) UnStored
-        ]
+    let
+        subs =
+            [ newData NewData
+            , newError NewError
+            , dataSaved DataSaved
+            , dataNotSaved DataNotSaved
+            ]
+    in
+        if model.modified then
+            Sub.batch (subs ++ [ Time.every (Time.millisecond * 200) Modifying ])
+        else
+            Sub.batch subs
```

Comment:
You can extract `Sub.batch` here, although I don't know if it's worth it.

It's really cool that you can subscribe to effects dynamically!